doc/source/index.rst

```diff
@@ -5,7 +5,9 @@ Galaxy is an open, web-based platform for accessible, reproducible, and transpar
 
 For more information on the Galaxy Project, please visit the https://galaxyproject.org
 
-The community maintains `a list of awesome Galaxy resources <https://github.com/galaxyproject/awesome-galaxy/>`__ related to all aspects of Galaxy.
+This site contains documentation for administrators and developers. For user documentation, please visit the `Galaxy Training Network <https://training.galaxyproject.org/>`__.
```

Comment:
```suggestion
This site contains documentation for administrators and developers. For user documentation, and admin and developer tutorials, please visit the `Galaxy Training Network <https://training.galaxyproject.org/>`__.
```

maybe something like this? technically the gtn isn't "documentation" only "tutorials", and with the inclusion of admin/dev *tutorials* as a distinct thing from docs, maybe it's good to call that out?

Reply:
Thanks, that sounds good to me.